cluster/mesos/docker/mesos-slave/bin/wrapdocker

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+# Copyright 2013-2014 Jérôme Petazzoni, Johan Haleby, lalyos, James Harris,
+#                     Michael Elsdörfer, Tony Hesjevik, Esben Haabendal,
+#                     Michael A. Smith
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
i've got code that kinda moves that way, maybe i should publish it....   but not really this PR.